juniper_official/Solutions/EX-Platform/PoE-KPI-Playbook.playbook

```diff
@@ -0,0 +1,19 @@
+/*
+ * Playbook contains 3 rules which monitors POE status , interface and controller 
+ * power consumption and notifies the status.
+ * 
+ * 1) Rule check-poe-total-power, checks POE Controller power consumption and raises 
+ *    alarm in case of anomalies.
+ * 2) Rule check-poe-operational-status, POE Interface Operational state and  
+ *    notifies when anomalies are found.
```

Comment:
s/when anomalies are found./when state is down./

Reply:
updated